pythonDiff/diff_env/bin/Activate.ps1

```diff
@@ -0,0 +1,247 @@
+<#
```

Comment:
Everything in diff_env is autogenerated for the virtual environment and not part of the code, I'd remove these from the commit. You can use the .gitignore file to exclude them automatically